pymc/step_methods/metropolis.py

```diff
@@ -140,22 +140,22 @@ def __init__(
 
         Parameters
         ----------
-        vars: list
+        vars : list
             List of value variables for sampler
-        S: standard deviation or covariance matrix
+        S : standard deviation or covariance matrix
```

Comment:
standard deviation or covariance matrix aren't types, this is mixing type with description. I think I have never used Metropolis, so my guess about behaviour here might be wrong but here is a proposal:

```
S : array_like with shape (N,) or (N, N), optional
```
Then in `proposal_dist` description:

```
Function that returns zero-mean deviates when parameterized with
`S` (and n). If `S` is one dimensional, it defaults to a normal, with `S` as its standard deviation;
If `S` is two dimensional, it defaults to a multivariate normal with `S` as its covariance matrix.
```

Reply:
I have changed this one based on your suggestions. I also changed `S` description to match.